modules/vulnerability-management-export-workloads.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_mod-docs-content-type: CONCEPT
+[id="vulnerability-management-export-workloads_{context}"]
+= Exporting workload vulnerabilities via the API
+
+[role="_abstract"]
+This section provides example use cases to export workload vulnerabilities using {product-title}. In this context workloads are composed of deployments and their associated images. The export makes use of the `/v1/export/vuln-mgmt/workloads` streaming API. It allows the combined export of deployments and images. The `images` payload contains the full vulnerability information. The output is streamed and has the schema:
```

Comment:
```suggestion
You can export workload vulnerabilities in {product-title} by using the API. 

For these examples, workloads are composed of deployments and their associated images. The export uses the `/v1/export/vuln-mgmt/workloads` streaming API. It allows the combined export of deployments and images. The `images` payload contains the full vulnerability information. The output is streamed and has the following schema:
```

Reply:
The abstract part just needs to be a simple description of the feature (It's for the doc publishing site to show as a "preview" kind of) so I split the paragraph.